Week-5/hasDuplicate.js

```diff
@@ -0,0 +1,13 @@
+const hasDuplicate = (arr) => {
```

Comment:
You can also do it by checking length of set and array

Reply:
made suggested change